### PR TITLE
react_goods-selector

### PR DESCRIPTION
**Pull request by Tetiana-Hishchak**
https://Tetiana-Hishchak.github.io/react_goods-selector/

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -15,71 +17,65 @@ export const goods = [
...
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+export const App: React.FC = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
+  const messege = selectedGood.length > 0 ? `${selectedGood} is selected`
+    : 'No goods selected';
+  // const buttonDel = document.getElementsByClassName('delete ml-3');
```
Comment (VitaliyBondarenko1982) [inline src/App.tsx:24]: remove redundant commented code

File: src/App.tsx [hunk 2]
```diff
@@ -15,71 +17,65 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+export const App: React.FC = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
+  const messege = selectedGood.length > 0 ? `${selectedGood} is selected`
```
Comment (VitaliyBondarenko1982) [inline src/App.tsx:22]:
```suggestion
  const message = selectedGood.length > 0 ? `${selectedGood} is selected`
```
typo

File: src/App.tsx [hunk 3]
```diff
@@ -15,71 +17,65 @@ export const goods = [
...
+                  className={cn('button', {
+                    ' is-info': selectedGood === good,
+                  })}
+                  onClick={
+                    selectedGood === good
+                      ? () => setSelectedGood('')
+                      : () => setSelectedGood(good)
+                  }
+                >
+                  {selectedGood === good ? '-' : '+'}
```
Comment (VitaliyBondarenko1982) [inline src/App.tsx:68]: found 4 places where you use 
selectedGood === good
consider to make variable for example `const isSelected = selectedGood === good`
and use in all places

**Review by VitaliyBondarenko1982 — CHANGES_REQUESTED** [review]
Good job
Needs some improvements.

**Review by mykhalenych — APPROVED** [review]

**Pull request closed without merging**